integration/sawtooth_integration/tests/test_basic_auth_proxy.py

```diff
@@ -68,16 +68,17 @@ def _assert_valid_authed_request(self, url):
         try:
             response = urlopen(request, context=context)
         except HTTPError as e:
+            LOGGER.error('An error occured while requesting {}'.format(url))
+
             try:
                 error = json.loads(e.file.read().decode())['error']
+                error_string = 'REST API Error: {} - {}:'.format(
+                    error['code'], error['title'])
             except json.decoder.JSONDecodeError:
-                raise e
-
-            LOGGER.error('REST API Error: {} - {}:'.format(
-                error['code'], error['title']))
-            LOGGER.error(error['message'])
+                error_string = 'HTTP Error: {} - {}'.format(e.code, e.msg)
 
-            raise e
+            LOGGER.error(error_string)
```

Comment:
Excellent question. I do not know enough about the details of what is getting logged where and how it is important.

My thinking was that the logs _might_ be getting stored somewhere different than the test output. If in _all_ cases you will get both the error logs and the test output, then it is redundant and I will remove it.